models/br_senado_dados_abertos/code/METADATA_UPDATE_RENAME.md

```
@@ -0,0 +1,39 @@
+# Metadata column update — normalization (staging + prod)
+
+The dbt models were normalized. Backend column metadata must match on BOTH the
+`staging` and `prod` backends. Do the SAME operations on each (run env="staging"
+first, then env="prod").
+
+## Renamed columns (old name → new name)
+| table | delete (old) | the new name is created by bulk_upsert |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)